public/old-i18n/i18n_module_ar.properties

```diff
@@ -127,6 +127,7 @@ extended_filters=Advanced filters
 clear_filters=Clear all filters
 table_layout_at_least_one_column=Select at least one column
 confirm=Confirm
+remove_users=Remove user(s)?
```

Comment:
For new translations, we should use the new i18n instead of the old one.

Also the dialog title can be ``Remove users`` without the question mark